templates/conf.py_t

```diff
@@ -114,7 +114,9 @@ todo_include_todos = {{ ext_todo }}
 %}
 # Theme is builtin. Just set the name
 html_theme = '{{ html_theme }}'
-
+{% elif html_theme == 'fossasia' %}
```

Comment:
Instead of doing a seperate check for fossasia, could we just check whether {{html_theme}} is in DOCPATH, and if it is there, use it. this would have the additional advantage that users would be able to place their own themes similarly.